docs/uml-class-diagrams/display01/test/DisplayGpsManagerImplementationTests/DisplayGpsManagerImplementationTests.puml

```diff
@@ -0,0 +1,12 @@
+@startuml DisplayGpsManagerImplementationTests
+
+package edu.oakland.display01.tests {
+    class DisplayGpsManagerImplementationTests {
+        + void receivedGpsSignalStrengths()
+        + void acceptableGpsSignalStrengths()
+        + void poorGpsSignalStrengths()
+        + 
```

Comment:
Where is the if statement in your code? I think that #92 currently looks like it does the signal check, but neither does null checking...